src/main/java/com/example/voucher/constant/ServiceType.java

```diff
@@ -0,0 +1,16 @@
+package com.example.voucher.constant;
+
+import java.util.Arrays;
+
+public enum ServiceType {
+
+    EXIT,
+    Voucher;
```

Comment:
Voucher도 대문자로 통일하시면 좋을 것 같습니다. 그리고 enum은 아래와 같은 형태로 하시는게 관리에 더 편하실 거에요 :) 
```suggestion
    EXIT,
    Voucher
    ;
```

Reply:
7398ba51634bec1888a4810d1ccabe1749cbb43a 커밋 피드백 반영 
- 대문자로 써야함을 인지하고 있었으나 꼼꼼함이 부족했던 것 같습니다. 고쳐보겠습니다.
- 세미콜론을 한칸 내리는 표기법이 값에 대한 가독성이 좋다고 판단하여 고쳐봤습니다. 가독성 이외에도 관리면에서 용이한 점이 있을까요??

Reply:
피드백 반영 
- 구두로 피드백을 진행했습니다.